src/complexity.cc

```diff
@@ -144,34 +132,34 @@ LeastSq MinimalLeastSq(const std::vector<int>& n,
       oLogN, oN, oNLogN, oNSquared, oNCubed };
 
     // Take o1 as default best fitting curve
-    best_fit = CalculateLeastSq(n, time, FittingCurve(o1));
+    best_fit = MinimalLeastSq(n, time, FittingCurve(o1));
     best_fit.complexity = o1;
 
     // Compute all possible fitting curves and stick to the best one
     for (const auto& fit : fit_curves) {
-      LeastSq current_fit = CalculateLeastSq(n, time, FittingCurve(fit));
+      LeastSq current_fit = MinimalLeastSq(n, time, FittingCurve(fit));
       if (current_fit.rms < best_fit.rms) {
         best_fit = current_fit;
         best_fit.complexity = fit;
       }
     }
   } else {
-    best_fit = CalculateLeastSq(n, time, FittingCurve(complexity));
+    best_fit = MinimalLeastSq(n, time, FittingCurve(complexity));
     best_fit.complexity = complexity;
   }
 
   return best_fit;
 }
 
 std::vector<BenchmarkReporter::Run> ComputeStats(
-    const std::vector<BenchmarkReporter::Run>& reports)
+  const std::vector<BenchmarkReporter::Run>& reports)
```

Comment:
please run this through clang-format with google style to ensure these indents are correct and not changing for the sake of it.